platform-jobs/samza/publish-pipeline/src/main/config/publish-pipeline.properties

```diff
@@ -183,3 +183,7 @@ content.streaming_enabled=__content_streaming_enabled__
 #Configuration added to handle large artifacts
 content.artifact.size.for_online=209715200
 
+kafka.topics.learning.graph=__env__.learning.graph.events
```

Comment:
Please add comment for the config property. Also add property in local.publish-pipeline.properties (only if the property is mandatory).

# Graph event topic config for pushing collection units 
kafka.topics.learning.graph=__env__.learning.graph.events

# For enabling pushing of units directly to ES
content.es.sync.enable=false

# Kafka urls configuration
kafka.urls=__graphevent_kafka_url__